### ISSUE TITLE
Please add BL3P exchange

### ISSUE THREAD

**Issue by roshii** [kroitor/ccxt#44]
I'd love to see subject exchange supported by your library, see reference on @BitonicNL 's [docs](https://github.com/BitonicNL/bl3p-api/blob/master/docs/base.md)

Cheers

**Comment by kroitor** [comment]
Thanks! Will integrate it ASAP )

**Comment by kroitor** [comment]
@roshii, we have implemented their API and we are now testing it, it should be available within a new version (release) very soon. It should appear in the docs after the tests pass (may take from minutes to some time if we experience problems along the way). 

We tested the public API, and it works fine. But we don't have the option to test their private API from where we are, because they have a very restricted list of supported countries (why would they), and our country isn't there. We contacted their support for a sandboxed pair of keys or something, still waiting for their reply.

In the meantime, @roshii, we would like to ask you for some collaboration with us to test their private API... Maybe, you could send us a pair of keys to bl3p for testing, or perform some tests on your side, if you don't mind, that would be very helpful from you...

**Comment by roshii** [comment]
wow, I'm impressed by you prompt integration :)
I'll see if I can share key pair and if not I'll run tests myself, I'll let you know early PM today

thanks

**Comment by roshii** [comment]
See here after, running `test.py bl3p`
I can't generate any sand-boxed key pair unfortunately, I've contacted them, let see.

In the meanwhile, let me know if I can do anything

```
[92mbl3p[0m
[92mbl3p[0m [92mBTC/EUR[0m fetching ticker...
[92mbl3p[0m [92mBTC/EUR[0m ticker 2017-07-24T14:29:09.000Z high: 2440.0 low: 2291.11 bid: 2402.0 ask: 2409.5 volume: 190.96422113
[92mbl3p[0m [92mBTC/EUR[0m fetching order book...
[92mbl3p[0m [92mBTC/EUR[0m order book 2017-07-24T14:29:21.000Z bid: 2402.0 bidVolume: 0.20583201 ask: 2409.5 askVolume: 0.80594566
Traceback (most recent call last):
  File "test.py", line 225, in <module>
    try_all_proxies (market, proxies)
  File "test.py", line 166, in try_all_proxies
    test_market (market)
  File "test.py", line 99, in test_market
    balance = market.fetch_balance ()
  File "C:\Users\roshii\Downloads\ccxt-master\ccxt\__init__.py", line 3582, in fetch_balance
    balance = response['wallets']
KeyError: 'wallets'
```

**Comment by roshii** [comment]
See here after when doing a `print(response)` just before line 3582

`
{'result': 'error', 'data': {'code': 'NOT_AUTHENTICATED', 'message': 'Could not validate call: Invalid signature'}}`

**Comment by kroitor** [comment]
> I can't generate any sand-boxed key pair unfortunately, I've contacted them, let see.

Don't worry, we obtained a pair of keys from them as of today, so we will test and fix for that very soon. Thank you for your involvement!

**Comment by kroitor** [comment]
Ok, we have it fixed now ) The private part will be available as of next version 1.1.73 (to be released in a matter of minutes)

**Comment by kroitor** [comment]
The fix is now available in version 1.1.73 in pypi/npm! Thanks!
